frontend/smart-kickers-game/src/apis/Game.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export function resetGame(gameId) {
+  return axios.post(`http://localhost:3006/reset/${gameId}`);
```

Comment:
We have only one game at the time and our local server runs on port 3000 so I suggest a small change here for now. 
```suggestion
  return axios.post(`http://localhost:3000/reset`);
```